test/Controller/ArticleControllerTest.php

```diff
@@ -575,8 +575,14 @@ public function it_displays_metrics()
         $this->assertSame(200, $client->getResponse()->getStatusCode());
         $this->assertSame('Article title', $crawler->filter('.content-header__title')->text());
 
-        $this->assertContains('1,234', $crawler->filter('.contextual-data__list_title:contains("Cited") + .contextual-data__list_desc')->text());
-        $this->assertContains('5,678', $crawler->filter('.contextual-data__list_title:contains("Views") + .contextual-data__list_desc')->text());
+        $this->assertSame(
+            [
+                'Cited 1,234',
+                'Views 5,678',
+                'Comments 0',
```

Comment:
Should this be 'Annotations', or is this Disqus-related?

Reply:
Disqus (there's existing tests for annotations).

Reply:
+1